Exercises/Functions/functions.js

```diff
@@ -2,6 +2,7 @@
 // In this exercise, you are going to write some more code related to preparing and cooking your brilliant lasagna from your favorite cookbook.
 // You have five tasks. The first one is related to the cooking itself, the other four are about the perfect preparation.
 
+
 // 1. Determine whether the lasagna is done
```

Comment:
'Const' instead of 'let' for functions
When calling 'preparationTime', put the second parameter too
For 'scaleRecipe', it's almost perfect, you just need newPortions = recipe * (portions / 2)  because the recipe is already made for 2 portions